debug_scripts/send_validator_logs.py

```diff
@@ -0,0 +1,63 @@
+#!/bin/python3
+import argparse
+import boto3
+import datetime
+import gzip
+import io
+import sys
+import urllib.parse
+
+
+def filter_log_file(log_file: str, start_time: datetime.datetime, end_time: datetime.datetime) -> io.BytesIO:
```

Comment:
nit: I'd actually have this function return list[str] (list of lines)

this would make testing a lot easier